prefixer/composer.json

```diff
@@ -18,7 +18,7 @@
     "symfony/polyfill-intl-normalizer": "1.18.1",
     "symfony/polyfill-php72": "1.18.1",
     "symfony/validator": "3.4.43",
-    "twig/twig": "2.12.5"
+    "twig/twig": "2.13.1"
```

Comment:
After updating twig I noticed that there are some files we could remove to keep plugin package smaller:
```
twig/twig/drupal_test.sh
twig/twig/README.srt
twig/twig/src/Test - directory
```
Also this https://github.com/mailpoet/mailpoet/blob/master/build.sh#L130 might be no longer needed.